File: backend/src/main/java/tom/projects/monkeylog/repository/workout/ExerciseRepository.java
package tom.projects.monkeylog.repository.workout;

import org.springframework.data.jpa.repository.JpaRepository;
import tom.projects.monkeylog.model.exercise.Exercise;

import java.util.List;
import java.util.UUID;

public interface ExerciseRepository extends JpaRepository<Exercise, UUID> {
    List<Exercise> findAllByUserId(UUID userId);
}

File: backend/src/main/kotlin/tom/com/monkeylog/repository/workout/ExerciseRepository.kt
package tom.com.monkeylog.repository.workout

import org.springframework.data.jpa.repository.JpaRepository
import tom.com.monkeylog.model.exercise.Exercise
import java.util.*

interface ExerciseRepository : JpaRepository<Exercise, UUID> {
    fun findAllByUserId(userId: UUID): List<Exercise>
}
